runtime/near-vm-runner-standalone/contract/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use borsh::{BorshDeserialize, BorshSerialize};
```

Comment:
Please rename folder to `example-contract`.

Reply:
Done.